miden-tx/tests/integration/scripts/swap.rs

```diff
@@ -68,14 +68,15 @@ fn prove_swap_script() {
     let tx_script_target = executor
         .compile_tx_script(tx_script_code.clone(), vec![(target_pub_key, target_sk_felt)], vec![])
         .unwrap();
+    let tx_args_target = TransactionArgs::new(Some(tx_script_target), None);
 
     // Execute the transaction
-    let executed_transaction = executor
-        .execute_transaction(target_account_id, block_ref, &note_ids, Some(tx_script_target))
+    let transaction_result = executor
```

Comment:
I don't think we need to change `executed_transaction` into `transaction_result` here and in other places in this file.